### DISCUSSION TITLE
How to overwrite AdminConfig parameters on standalone Topic Operator?

### DISCUSSION THREAD

**Discussion by jozenstar** [strimzi/strimzi-kafka-operator#4787] [Q&A]
Hi, I'm trying to overwrite some configs of standalone Topic Operator deployment by specifying ex. `-Dparam.name=value` in   `STRIMZI_JAVA_SYSTEM_PROPERTIES` env var. However, in the logs I can see that the value remains the same. 
Is there an option to overwrite AdminConfig values?

**Comment by scholzj** [comment]
I do not think the Kafka Admin API has any support for reading configuration options from the system properties. And the Topic Operator definitely does not add anything like that. So I do not think this is possible.

**Reply by jozenstar** [reply]
Thanks!
I've made a fix for that
https://github.com/strimzi/strimzi-kafka-operator/pull/4796
